projects/admin/src/app/record/detail-view/item-detail-view/item-detail-view.component.ts

```diff
@@ -115,6 +117,7 @@ export class ItemDetailViewComponent implements DetailRecord, OnInit, OnDestroy
    * @param _userService - UserService
    */
   constructor(
+    public itemApiService: ItemApiService,
```

Comment:
Is it a good practice. Do we need to create a `private` property and expose this using a getter?